### DISCUSSION TITLE
New file in the RGraph library - do I need to do anything?

### DISCUSSION THREAD

**Discussion by heyesr** [cdnjs/cdnjs#14256] [General] [answered]
Hi,

If I add a new library file to my RGraph library - and call it RGraph.svg.common.zoom.js - do I need to do anything? After reading the conf file for RGraph ( https://github.com/cdnjs/packages/blob/master/packages/r/RGraph.json ) I would guess that I don't and anything that fits the pattern RGraph.*.js is mirrored by yourselves. Is that correct?

Thanks!

**Comment by MattIPv4** [comment, marked as answer]
That config will pull in any file matching the name `RGraph.*.js` within the `libraries` directory. So, yes, a file named `RGraph.svg.common.zoom.js` should match assuming it is in `libraries`.

**Reply by heyesr** [reply]
Thanks for the response!
